external-import/mandiant/src/connector/__main__.py

```diff
@@ -7,6 +7,7 @@
     mandiantConnector = Mandiant()
     while True:
         mandiantConnector.run()
-except Exception:
+except Exception as err:
+    print(err)
     time.sleep(10)
     sys.exit(0)
```

Comment:
Hi, shouldn't we either raise/not catch the exception in order to exit with a non-zero status code ?

That will help monitor critical errors of this connector leading to unexpected shutdowns.